src/actionCreators/qa.js

```diff
@@ -0,0 +1,75 @@
+// Copyright 2019 Stanford University see LICENSE for license
+
+import { loadingQaResults, qaResultsReceived } from 'actions/index'
+import Swagger from 'swagger-client'
+import swaggerSpec from 'lib/apidoc.json'
+import { getLookupConfigItems } from 'utilities/propertyTemplates'
+import Config from 'Config'
+
+const search = (query, propertyTemplate) => (dispatch) => {
+  dispatch(loadingQaResults())
+
+  return getSearchResults(query, propertyTemplate).then((values) => {
+    dispatch(qaResultsReceived(values))
+  })
+}
+
+const getSearchResults = (query, propertyTemplate) => Swagger({ spec: swaggerSpec })
+  .then((client) => {
+    const lookupConfigs = getLookupConfigItems(propertyTemplate)
+
+    // Create array of promises based on the lookup config array that is sent in
+    const lookupPromises = createLookupPromises(client, query, lookupConfigs)
+
+    /*
+     * If undefined, add info - note if error, error object returned in object
```

Comment:
If defined?

Reply:
I have no idea. This wasn't something I added, just moved from https://github.com/LD4P/sinopia_editor/pull/1161/files#diff-93b04186261f1a6fa9e8f243d588ef94L199